lib/main_common.pm

```diff
@@ -1598,6 +1599,42 @@ sub load_filesystem_tests {
     loadtest 'console/snapper_used_space' if (is_sle('15-SP1+') || (is_opensuse && !is_leap('<15.1')));
 }
 
+sub get_wicked_tests {
+    if (check_var('WICKED', 'advanced')) {
```

Comment:
I was playing around with perl and found an alternative to list all the test cases here to save some lines of code, but I'm not sure if it's worth it. It would be similar to this:

    my $dir = '/var/lib/openqa/share/tests/sle/tests/wicked/' . get_var('WICKED') . '/sut';
    my @test_array;
    opendir(DIR, $dir) or die $!;
    while (my $file = readdir(DIR)) {
        next if ($file =~ m/^\./);
        (my $test_name = $file) =~ s/\.[^.]+$//;
        push @test_array, $test_name;
    }
    closedir(DIR);
    return @test_array;

Reply:
I like this approach. But we should find the scripts similar as loadtest() does (https://github.com/os-autoinst/os-autoinst/blob/master/autotest.pm#L50). So we do not get in trouble with other global configuration parameters to the worker.
The only thing I would add is a WICKED_EXCLUDE variable, so if you test one test-case, you have the possibility to exclude all the others.

Reply:
agree @cfconrad , not good idea to hard code path where tests are. I have idea to make it flexible in nearest future :)

Reply:
sure, that is a draft and just the idea, it's hard-coded but for sure we should follow same approach as loadtest. But let's leave it for now, at least for this PR as it's not so important.